default.nix

```diff
@@ -0,0 +1,5 @@
+with import <nixpkgs> {};
```

Comment:
Please add a component describing what this is.
Something like this I suppose:
# Package description for Nix (http://nixos.org)